sidebars.js

```diff
@@ -15,162 +15,84 @@
           ],
         },
         {
           type: 'category',
-          label: 'Create Wasm Wrappers',
+          label: 'Create Wraps',
           items: [
             {
               type: 'category',
               label: 'Tutorial',
               items: [
-                'tutorials/create-wasm-wrappers/tutorial/project-setup',
-                'tutorials/create-wasm-wrappers/tutorial/project-folder',
-                'tutorials/create-wasm-wrappers/tutorial/build-deploy-test',
-                'tutorials/create-wasm-wrappers/tutorial/adding-new-methods',
-                'tutorials/create-wasm-wrappers/tutorial/writing-tests-with-workflows',
-                'tutorials/create-wasm-wrappers/tutorial/deploy-locally-and-test',
+                'tutorials/create-wraps/tutorial/project-setup',
+                'tutorials/create-wraps/tutorial/project-folder',
+                'tutorials/create-wraps/tutorial/build-deploy-test',
+                'tutorials/create-wraps/tutorial/adding-new-methods',
+                'tutorials/create-wraps/tutorial/writing-tests-with-workflows',
+                'tutorials/create-wraps/tutorial/deploy-locally-and-test',
               ]
             },
-            'tutorials/create-wasm-wrappers/polywrap-manifest',
-            'tutorials/create-wasm-wrappers/default-plugins'
+            'tutorials/create-wraps/polywrap-manifest',
+            'tutorials/create-wraps/default-plugins'
           ],
         },
         {
           type: 'category',
-          label: 'Testing Wrappers',
+          label: 'Test Wraps',
           items: [
-            'tutorials/testing-wrappers/in-typescript',
-            'tutorials/testing-wrappers/infra-pipeline',
+            'tutorials/test-wraps/in-typescript',
+            'tutorials/test-wraps/infra-pipeline',
           ],
         },
         {
           type: 'category',
-          label: 'Build & Deploy Wasm Wrappers',
+          label: 'Build & Deploy Wraps',
           items: [
-            'tutorials/build-and-deploy-wasm-wrappers/build-pipeline',
-            'tutorials/build-and-deploy-wasm-wrappers/deploy-to-ipfs',
-            'tutorials/build-and-deploy-wasm-wrappers/publish-to-ens',
-            'tutorials/build-and-deploy-wasm-wrappers/deploy-pipeline',
+            'tutorials/build-and-deploy-wraps/build-pipeline',
+            'tutorials/build-and-deploy-wraps/deploy-to-ipfs',
+            'tutorials/build-and-deploy-wraps/publish-to-ens',
+            'tutorials/build-and-deploy-wraps/deploy-pipeline',
           ],
         },
         {
           type: 'category',
-          label: 'Create Plugin Wrappers',
+          label: 'Create Plugin Wraps',
           items: [
-            'tutorials/create-plugin-wrappers/create-js-plugin',
-            'tutorials/create-plugin-wrappers/plugin-manifest',
-            'tutorials/create-plugin-wrappers/plugin-architecture'
+            'tutorials/create-plugin-wraps/create-js-plugin',
```

Comment:
Are we still calling these "plugin wraps" or should they just be "plugins"?

Reply:
Will be called Plugins 😄